mappings/net/minecraft/entity/projectile/ExplosiveProjectileEntity.mapping

```diff
@@ -6,10 +6,21 @@ CLASS awc net/minecraft/entity/projectile/ExplosiveProjectileEntity
 	FIELD f life I
 	FIELD g ticks I
 	METHOD <init> (Laij;DDDDDDLbhh;)V
-		ARG 1 world
+		ARG 1 type
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 velocityX
+		ARG 10 velocityY
+		ARG 12 velocityZ
+		ARG 14 world
 	METHOD <init> (Laij;Laio;DDDLbhh;)V
-		ARG 1 world
+		ARG 1 type
 		ARG 2 owner
+		ARG 3 velocityX
+		ARG 5 velocityY
+		ARG 7 velocityZ
```

Comment:
They are normalized in the constructor, so `velocity` might be not quite right here. However, a random spray value gets added to them before that, so a higher velocity will increase the accuracy, however the actual speed is not affected.

Reply:
`directionVec`?

Reply:
I thought about that too, wasn't sure because of the accuracy change, though.